base/dict.jl

```diff
@@ -135,10 +135,10 @@ copy(d::Dict) = Dict(d)
 
 const AnyDict = Dict{Any,Any}
 
+Dict(ps::Pair...)                            = Dict{Any,Any}(ps) # define first, since method sort order is wrong for the following
```

Comment:
Just curious, can you explain why the sort order is wrong when this is below?

Presumably, `DataStructures.jl` will need to make similar changes (e.g., at https://github.com/JuliaCollections/DataStructures.jl/blob/master/src/default_dict.jl#L88-L96)

Reply:
Bump, @vtjnash.

Reply:
Probably fixed now—Jeff has made a lot of changes to method sort order since then